### ISSUE TITLE
utils module incorrectly resolved as lightkurve.seismology.utils

### ISSUE THREAD

**Issue by orionlee** [KeplerGO/lightkurve#605]
#### Problem description
Functions / Classes in `lightkurve.utils` module are somehow incorrectly resolved as `lightkurve.seismology.utils`, thus raising errors

#### Example

```python
# Assume lc is a LightCurve object

# failed with: 
#  AttributeError: module 'lightkurve.seismology.utils' has no attribute 'btjd_to_astropy_time'
import lightkurve as lk
lk.utils.btjd_to_astropy_time(lc.time)

# failed with: 
#  AttributeError: module 'lightkurve.seismology.utils' has no attribute 'btjd_to_astropy_time'
import lightkurve.utils as lkutils
lkutils.btjd_to_astropy_time(lc.time)

# see where the module is resolved
lk.utils
<module 'lightkurve.seismology.utils' from '[Anacoda-Base]\\lib\\site-packages\\lightkurve\\seismology\\utils.py'>

# somehow using from ... import ... works
from lightkurve.utils import  btjd_to_astropy_time
btjd_to_astropy_time(lc.time)
```

#### Expected behavior
Normal imports should work.

#### Environment

-  platform: Windows, Jupyter Notebook (Python 3.7)
-  lightkurve version (e.g. 1.0b6): 1.2.0
-  installation method (e.g. pip, conda, source): conda-forge

**Comment by barentsen** [comment]
Oops!  I confirm this is a bug.  Thank you for reporting @orionlee! :+1::+1::+1:

**Comment by barentsen** [comment]
Thanks again for your report @orionlee!  The issue has been resolved on the master branch (#606) and the fix will be included in the upcoming v1.3 release of Lightkurve!

**Comment by orionlee** [comment]
Thanks for the speedy response @barentsen
